index.js

```diff
@@ -186,8 +186,18 @@ exports.viewAccount = async function (options) {
     if (state && state.amount) {
         state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
     }
-    console.log(`Account ${options.accountId}`);
-    console.log(inspectResponse.formatResponse(state));
+    const storageInM=state.storage_usage/1e+6;
+    const accountDetails={
+        'Account:':`${options.accountId} at block ${state.block_height} (https://explorer.testnet.near.org/blocks/${state.block_hash})`,
+        [chalk.bold('amount:')]:`${state.amount} ${chalk.bold('yn')}`,
+        'locked:':state.locked,
+        'storage:':`${storageInM}M (${state.storage_usage} bytes)`,
+        'code_hash:':`'${state.code_hash}'`
```

Comment:
Let's make sure that all the keys are **bold** and human-readable (**Code hash**, not `code_hash`)